include/GL/glui.h

```diff
@@ -34,11 +34,23 @@
 #ifndef GLUI_GLUI_H
 #define GLUI_GLUI_H
 
-// Having stdlib here first fixes some 'exit() redefined' errors on MSVC.NET
-// that come from old GLUT headers.
-#include <cstdlib>
+//Note: GCC is not required for this feature. It's just an indicator. 
+#ifdef __GNUC__
+//GLUI_GLX uses glXGetCurrentDisplay to find GLUT's chosen X display.
+//It will #include GL/glx.h and X11/Xlib.h. This covers all desktops.
```

Comment:
Maybe. But don't look at me. Mac OS has Xlib and GLX headers. They must do something. This (https://github.com/alecjacobson/glut) seems to be Apple's GLUT. Perhaps open-sourced and enhanced. Does it work with GLUI on Cocoa? Someone with Apple desktops should know.